holoviews/plotting/util.py

```diff
@@ -70,6 +72,19 @@ def collate(obj):
         raise Exception(undisplayable_info(obj))
 
 
+def initialize_dynamic(obj):
+    """
+    Initializes all DynamicMap objects contained by the object
+    """
+    dmaps = obj.traverse(lambda x: x, specs=[DynamicMap])
+    for dmap in dmaps:
+        if dmap.sampled:
+            # Skip initialization until plotting code
```

Comment:
I don't think there is any explicit initialization in the plotting code it just works because ``select`` is called requesting the first sample defined by other HoloMap(s).